packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -255,37 +261,58 @@ class VifAllowedIps extends BaseComponent {
         )}
         <Row>
           <Col size={10}>
-            {showNewIpForm ? (
-              <span onBlur={this._toggleNewIp}>
+            {showIpSelector && (
+              <span onBlur={this._hideIpInputs}>
                 {resourceSet ? (
                   <SelectResourceSetIp
                     autoFocus
                     containerPredicate={this._getIsNetworkAllowed()}
                     onChange={this._addIp}
                     predicate={this._getIpPredicate()}
                     resourceSetId={resourceSet}
+                    value={null}
```

Comment:
Why?

Reply:
To keep the selector empty since I'm keeping it visible after a `onChange`, now.